app/config/parameters.yml.no-env

```diff
@@ -27,6 +27,9 @@ parameters:
     # to enable the comments feature you should pass 'always-active' instead of 'inactive'
     env(COMMENTS_FEATURE): inactive
     env(CREATE_ACCOUNT_FEATURE): inactive
+    env(USE_MAIL_FEATURE): inactive
+
+    env(MAIL_FEATURE_FROM_EMAIL): send@example.com
```

Comment:
The default values should be usable.  `send@example.com` is not.
Probably should be set to ~ (representing null).